apps/ui/pages/application/[...params].tsx

```diff
@@ -221,9 +221,18 @@ function ApplicationRootPage({ slug, data }: PropsNarrowed): JSX.Element {
   const errorTranslated =
     errorMessage !== "" ? t(`errors:applicationMutation.${errorMessage}`) : "";
 
+  useEffect(() => {
+    if (errorTranslated !== "") {
+      toast({
+        label: t("common:error.error"),
```

Comment:
label necessary?

Reply:
label removed, and switched to use helper function